typescript/packages/jumble/src/components/NavPath.tsx

```diff
@@ -18,7 +18,7 @@ export function NavPath({ replicaId, charmId }: NavPathProps) {
       if (charmId) {
         const charm = await charmManager.get(charmId);
         if (charm) {
-          setCharmName(charm.get()[NAME]);
+          setCharmName(charm.getAsQueryResult()[NAME]);
```

Comment:
this works, but `charm.getAsCell().key(NAME).get()` would be using the new cells, and in principle you could also do `.sink(setCharmName)` instead of `.get()`, if React allows calling `setCharmName` at any time.

(soon I want to switch over `run`, etc. to return cells directly).